e2e/cypress/integration/search/results_post_spec.js

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+describe('Search', () => {
+    before(() => {
+        // # Login and navigate to the app
+        cy.apiLogin('user-1');
+        cy.visit('/');
+    });
+
+    it('S19944 Highlighting does not change to what is being typed in the search input box', () => {
+        const apple = `apple${Date.now()}`;
+        const banana = `banana${Date.now()}`;
+
+        const message = apple + ' ' + banana;
+
+        // # Post a message as "apple" and "banana"
+        cy.postMessage(message);
+
+        // # Search for "apple"
+        cy.get('#searchBox').type(apple).type('{enter}');
+
+        // # Get last postId
+        cy.getLastPostId().as('lastPostId');
+
+        // * Search result should return one post with highlight on apple
+        cy.get('@lastPostId').then((postId) => {
+            verifySearchResult(1, postId, message, apple);
+        });
+
+        // * Type banana on search box but don't hit search
+        cy.get('#searchBox').clear().type(banana);
```

Comment:
The test fails at this step because the input box does not has focus. Can you please check and add ```{force: true}```

![Screenshot 2019-11-11 at 11 04 50 PM](https://user-images.githubusercontent.com/1429138/68608089-16c9ea80-04d8-11ea-8d3a-7d053ac4f224.png)

Reply:
Hmm, it's not happening on me.  I've added visibility check.  Could you please check again?  Thanks